### PR TITLE
Sample project can now redirect to HTTPS

### PR DESCRIPTION
**Pull request by mikaelm12**
@Tratcher @danroth27 @natemcmaster @muratg

### CODE REVIEW

File: test/Microsoft.AspNetCore.Rewrite.Tests/UrlActions/ForbiddenActionTests.cs [hunk 1]
```diff
@@ -12,14 +12,14 @@ public class ForbiddenActionTests
...
         {
-            // Arrange
+
             var context = new RewriteContext {HttpContext = new DefaultHttpContext()};
             var action = new ForbiddenAction();
 
-            // Act
+
             action.ApplyAction(context, null, null);
-
```
Comment (natemcmaster) [inline test/Microsoft.AspNetCore.Rewrite.Tests/UrlActions/ForbiddenActionTests.cs]: Nit: remove empty lines or leave comments...doesn't matter either way to me.

**Comment by natemcmaster** [comment]
Code looks :shipit:

Just curious: is the testCert.pfx going to make chrome show the big red scary "do not enter" warnings?

File: samples/RewriteSample/project.json [hunk 2]
```diff
@@ -2,7 +2,8 @@
   "version": "1.1.0-*",
   "dependencies": {
     "Microsoft.AspNetCore.Rewrite": "1.1.0-*",
-    "Microsoft.AspNetCore.Server.Kestrel": "1.1.0-*"
+    "Microsoft.AspNetCore.Server.Kestrel": "1.1.0-*",
+    "Microsoft.AspNetCore.Server.Kestrel.Https": "1.0.0"
```
Comment (Tratcher) [inline samples/RewriteSample/project.json]: 1.1.0-*

**Comment by mikaelm12** [comment]
Yeah @natemcmaster . All the warnings

**Comment by natemcmaster** [comment]
@mikaelm12 don't forget to cleanup unused branches when you're done :)

**Pull request merged**